filament/backend/src/metal/MetalDriver.mm

```diff
@@ -934,9 +952,12 @@
             srcTarget->height - (NSUInteger) srcRect.bottom - srcRect.height,
             srcRect.width, srcRect.height);
 
+    const NSInteger dstHeight =
+            dstTarget->isDefaultRenderTarget() ?
+            mContext->currentSurface->getSurfaceHeight() : dstTarget->height;
     MTLRegion dstRegion = MTLRegionMake2D(
             (NSUInteger) dstRect.left,
-            dstTarget->height - (NSUInteger) dstRect.bottom - dstRect.height,
+            dstHeight - (NSUInteger) dstRect.bottom - dstRect.height,
```

Comment:
This also seems unrelated?

Reply:
Technically unrelated, but this bug was "exposed" by framebuffer fetch.